### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by BouaikIlham**
- [ ] add linter
- [ ] used correct [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md):
- [ ] README file
- [ ] Pull Request is made from 'feature branch' into the main branch.
- [ ] add "Works" section
- [ ] add the "About myself" section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,9 +43,176 @@ <h3 class="h3">LET’S CONNECT</h3>
...
+              sign-ups required.</p>
+          </div>
+          <div class="languages-list-container">
+            <ul class="lang-container">
+              <li class="langage-item"><span>JavaScript</span></li>
+              <li class="langage-item"><span>HTML</span></li>
+              <li class="langage-item"><span>CSS</span></li>
+            </ul>
+          </div>
+          <button type="button" class="see-btn">See Project</button>
```
Comment (akucintavalent) [inline index.html]: Please wrap each project card into its own container. We will need all card elements to be grouped into separate cards later on. You should apply `display: flexbox` to each project card. Kindly wrap all cards (`div` elements that represent cards) into one container and use `display: grid` CSS property to align the cards.

File: index.html [hunk 2]
```diff
@@ -43,9 +43,176 @@ <h3 class="h3">LET’S CONNECT</h3>
...
```
Comment (akucintavalent) [inline index.html:99]: Your buttons should have different styles applied when in a normal state, when you hover over them, and when they are pressed. Kindly follow the Figma template.

image

File: index.html [hunk 3]
```diff
@@ -43,9 +43,176 @@ <h3 class="h3">LET’S CONNECT</h3>
...
+        <h4>JavaScript</h4>
+      </li>
+      <li class="Languges-icon">
+        <img class="ellipse" src="images/Ellipse1(1).jpg" alt="">
+        <h4>HTML</h4>
+      </li>
+      <li class="Languges-icon"> 
+        <img class="ellipse" src="images/Ellipse1(2).jpg" alt="">
+        <h4>CSS</h4>
+      </li>
```
Comment (akucintavalent) [inline index.html]: Kindly use the proper fonts  in this section like in the template:

image

image

File: index.html [hunk 4]
```diff
@@ -43,9 +43,176 @@ <h3 class="h3">LET’S CONNECT</h3>
...
+        <h2 class="names">Frameworks</h2>
+        <img class="bottom" src="images/arrow_right.png" alt="arrow_bottom">
+      </li>
+    </ul>
+    <ul>
+      <li class="Languages-arrow-bottom"> 
+        <h2 class="names">Skills</h2>
+        <img class="bottom" src="images/arrow_right.png" alt="arrow_bottom">
+      </li>
+    </ul>
```
Comment (akucintavalent) [inline index.html]: The technology categories ("Languages", "Frameworks" and "Skills") should in one `ul`, each of them in a `li`

File: index.html [hunk 5]
```diff
@@ -43,9 +43,176 @@ <h3 class="h3">LET’S CONNECT</h3>
...
+              <li class="langage-item"><span>HTML</span></li>
+              <li class="langage-item"><span>CSS</span></li>
+            </ul>
+          </div>
+          <button type="button" class="see-btn">See Project</button>
+        </li>
       </ul>
     </div>
   </section>
+  <section class="about-me">
```
Comment (akucintavalent) [inline index.html]: Please align the content of the section with flexbox.

File: index.html [hunk 6]
```diff
@@ -12,7 +12,7 @@
 
 <body>
   <nav class="navbar">
-    <h2>My Logo</h2>
+    <h2 class="mylogo">My Logo</h2>
     <img class="icon" src="images/Union.jpg" alt="Union">
   </nav>
   <section class="section">
```
Comment (akucintavalent) [inline index.html]: The background of the greeting sections should be write according to the Figma template.

Yours:
image

Figma's
image

File: index.html (hunk 1 above)
Comment (akucintavalent) [inline index.html]: Each card should have white background:

Yours:
image

Figma's:
image

File: index.html (hunk 5 above)
Comment (akucintavalent) [inline index.html]: About me section should have white background. And the space between My works section and About me section should look like this:

image

File: index.html (hunk 3 above)
Comment (akucintavalent) [inline index.html]: The technologies categories should look like this:

image


Your design: 

image

**Review by akucintavalent — CHANGES_REQUESTED** [review]
Hi there, @BouaikIlham 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight :tada: :champagne: 
- Nice README file :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
- [ ] Please always try to preserve the distance between elements like the template does.
Zrzut ekranu 2022-03-22 o 23 41 42

- [ ] Kindly do your best to follow the Figma design as much as possible.
- [ ] Kindly checkout Chrome Dev Tools and try to check how your page looks on different mobile devices. Try to always check how your page looks on different mobile devices and try to make it as responsive as possible.
- [ ] [OPTIONAL] In your README file, please replace `cd Hello-Microverse` with `cd Portfolio-project`. Please provide information on what exactly the user has to open and how. The phrase "using live server" could be not understandable to some people. 
image
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (AnselemOdims) [reply index.html:99]: - [x] Please kindly implement this requested change

File: index.html [hunk 7]
```diff
@@ -11,40 +11,251 @@
...
+      <li><a href="#"><img src="images/Vector.svg" alt="thumb" /></a></li>
+    </ul>
+  </section>
+
+  <section>
+    <div class="cards-container">
+      <ul class="cards-list flex-center-culomn">
+        <li class="card-item flex-center-culomn">
+          <img class="card-image" src="images/SnapshootPortfolio.jpg" alt="Portfolio Portfolio">
+          <div class="list-conatiner">
```
Comment (AnselemOdims) [inline index.html]: - [ ] As already stated by the previous reviewer, please ensure there is enough spacing between the headline section and the works section

### Your Design 
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @BouaikIlham , 
Good job implementing some of the changes requested by the previous reviewer but you still have a bit to go.

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by BouaikIlham** [comment]
@AnselemOdims I don't understand this:
 Please kindly implement this requested change 
Can you explain clearly, please?

**Review by t-yanick — APPROVED** [review]
Hi @BouaikIlham, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**